### DISCUSSION TITLE
add a dev-latest-tensorrt tag to the builds?

### DISCUSSION THREAD

**Discussion by vajonam** [blakeblackshear/frigate#8201] [Ideas]
is it possible to add a 'dev-tensorrt' or 'dev-latest-tensorrt' tag to most recent build? allows us who want to be on the edge of the current beta cycle to be on the edge.

**Comment by NickM-27** [comment]
at one time there was an automatic beta build but people complained of breaking changes and used it without understanding it would not be stable. 

this is specifically not done, users should be aware of what the changes are in the commits when pulling a new dev image

this has been discussed many times, https://github.com/blakeblackshear/frigate/pull/6727 is one such example
